manifests/init.pp

```diff
@@ -36,15 +36,18 @@
 # $pulp_max_speed::                     The maximum download speed per second for a Pulp task, such as a sync. (e.g. "4 Kb" (Uses SI KB), 4MB, or 1GB" )
 #                                       type:Optional[String]
 #
-# $reverse_proxy::                      Add reverse proxy to the parent
-#                                       type:Boolean
+# $pulp_ports::                         Ports for the Pulp apache vhost to listen on
+#                                       type:Array[Integer[0, 65535]]
 #
-# $reverse_proxy_port::                 Reverse proxy listening port
-#                                       type:Integer[0, 65535]
+# $rhsm_proxy::                         Add a proxy rule for Red Hat Subscription Manager to the Pulp vhost
```

Comment:
I'm not sure this makes sense as an optional parameter with how we deploy configuration currently. Unless you use pure HTTP based Pulp repositories that you configure access to yourself, we have to couple deploying the RHSM proxy with having Pulp deployed to a proxy. Another way to look at it is, the katello-consumer RPM deployed to the proxy is deployed with this assumption.